pynsot/app.py

```diff
@@ -673,7 +680,7 @@ def list(self, data, display_fields=None, resource=None,
 
             if objects:
                 if grep:
-                    self.print_grep(objects)
+                    self.print_grep(objects, display_fields)
```

Comment:
You added `display_fields` to this function but I don't see it get used at all. Did you mean to filter the fields that get displayed with this list?

If that was your intention, I feel like `verbose_fields` may be more appropriate. That kwarg _should_ cover all fields on the object that are intended for humans.

If filtering display fields in grep output wasn't your intention, then I suggest removing `display_fields` from the function signature.

Reply:
Should probably just take it out. We had discussed this briefly last week to just display ALL THE FIELDS for each object, and you can see that represented in the implementation and the tests.

Reply:
Leaving this in was a total mistake -- I'll remove it ;) Thank you both!